content/modules/userprofile-query-android/pages/userprofile_query.adoc

```diff
@@ -187,6 +193,7 @@ An example of a document would be
 }
 ----
 
+[#lbl-usrprof]
```

Comment:
@ibsoln  Is there a reason for the "lbl" prefix? Would recommend something that is intuitive...Also suggest avoiding abbreviated forms of "user" to "usr" (its easy to get these wrong). This is a general comment and applies to all instances

Reply:
When I add explicit anchors like this I always use the prefixes fig-, tbl- or lbl- to indicate what the anchor relates to, a figure, table or general text label. It's just part of the invisible .adoc infrastructure and means the label can persist through  caption or heading text changes which might otherwise break external links.

Reply:
fixed here 0e822d8ca814ee03e5dbef5c369076128cd50036